packages/react-internal/src/compat/components/Button/ActionButton/ActionButton.styles.ts

```diff
@@ -15,14 +15,13 @@ export const getStyles = memoizeFunction(
         height: DEFAULT_BUTTON_HEIGHT,
         color: theme.palette.neutralPrimary,
         backgroundColor: 'transparent',
-        border: '1px solid transparent',
+        border: 'none',
```

Comment:
I looked at the history of why the transparent border was added, and it was done in #7538 to fix issues with the ActionButton's focus outline being clipped. However I tested the focus outline in the [PR deploy site](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/16188/merge/react/storybook/index.html?path=/story/button--button-action-example) for this change and it still works. So unless @khmakoto knows of some reason to keep the transparent border I think this change is okay.

Reply:
Hmm actually @khmakoto do you know if removing the border could cause any layout changes? (To be clear I'm in favor of removing the border if possible, but because Button is terrible I also want to make sure we don't break something.)

Reply:
Yes, I worry about layout changes, I think forcing a transparent border even in high contrast via `forcedColorAdjust` might be the solution here.

Reply:
You're right! For sure, that sounds like a great idea! Let's go with that.

Reply:
The newest iteration used `borderColor: 'Window'` on the ActionButton root as agreed. This caused an issue with the "Show code" toggle button on ExampleCard since it uses ActionButton. The border on this button disappears in high contrast mode. Even though its an ActionButton, I am assuming that we want a border around it since it has specific styling to show a border outside of high contrast. I added styling so that it would also show a border in high contrast.